ashley/src/test/kotlin/ktx/ashley/ComponentMappersSpec.kt

```diff
@@ -0,0 +1,17 @@
+package ktx.ashley
+
+import com.badlogic.ashley.core.Entity
+import org.assertj.core.api.Assertions.assertThat
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+
+object ComponentMappersSpec: Spek({
+  describe("utilities for component mappers") {
+    val entity = Entity()
+    it("should return a component mapper via a reified type") {
+      val mapper = mapperFor<Texture>()
```

Comment:
Does the mapper expose any properties (like wrapped class) that would could validate in the test? This test does not really check if the mapper is valid.